### PR TITLE
fix(Table): Remove pf-c-check__input class from the check-all input

### LINKED ISSUE #1271: Table component - Select All checkbox
The Table component has the following html and attributes for the Select All checkbox:
```
<td class="pf-c-table__check">
  <input class="pf-c-check__input" type="checkbox" name="check-all" aria-label="Select all rows">
</td>
```

In speaking with @mcoker, he had mentioned that the class `pf-c-check__input` should not be used here, and should be removed.

### PR DESCRIPTION
**Pull request by boaz0**
closes #1271 

//cc @jgiardino @mcoker

### CODE REVIEW

**Comment by mcoker** [comment]
thanks @boaz0! I don't think any of the checkbox inputs in the table need to use the `.pf-c-check` component's element `.pf-c-check__input`. If you look at the [check component](https://pf4.patternfly.org/components/Check/examples/), you'll see that `.pf-c-check`, `.pf-c-check__input`, and `.pf-c-check__label` are all required. That component is used to create a check/label combo, and has specific positioning for the `.pf-c-check__input` element for use with that component when the check is next to a label.

**Comment by boaz0** [comment]
@mocker I updated the commit. Feel free to review it and let me know if I'm missing something. Thanks. :dog:

File: src/patternfly/components/Table/examples/table-width-example.hbs [hunk 1]
```diff
@@ -25,7 +25,7 @@
   {{#> table-tbody}}
     {{#> table-tr}}
       {{#> table-td table-td--check="true"}}
-        {{#> check-input check-input--attribute='type="checkbox" name="checkrow1" aria-labelledby="width-node1"'}}{{/check-input}}
+        <input type="checkbox" name="checkrow1" aria-label="width-node1">
```
Comment (mcoker) [inline src/patternfly/components/Table/examples/table-width-example.hbs]:
```suggestion
        <input type="checkbox" name="checkrow1" aria-labelledby ="width-node1">
```

File: src/patternfly/components/Table/examples/table-width-example.hbs [hunk 2]
```diff
@@ -46,7 +46,7 @@
 
     {{#> table-tr}}
       {{#> table-td table-td--check="true"}}
-        {{#> check-input check-input--attribute='type="checkbox" name="checkrow2" aria-labelledby="width-node2"'}}{{/check-input}}
+        <input type="checkbox" name="checkrow2" aria-label="width-node2">
```
Comment (mcoker) [inline src/patternfly/components/Table/examples/table-width-example.hbs]:
```suggestion
        <input type="checkbox" name="checkrow2" aria-labelledby ="width-node2">
```

File: src/patternfly/components/Table/examples/table-expandable-example.hbs [hunk 3]
```diff
@@ -23,7 +23,7 @@
     {{#> table-tr table-tr--expanded="true"}}
       {{#> table-td table-td--toggle="true" table-td--button--attribute=(concat 'aria-labelledby="' table--id '-node1 expandable-toggle1" id="expandable-toggle1" aria-label="Details" aria-controls="' table--id '-content1"')}}{{/table-td}}
       {{#> table-td table-td--check="true"}}
-        {{#> check-input check-input--attribute=(concat 'type="checkbox" name="checkrow1" aria-labelledby="' table--id '-node1" aria-label="Details"')}}{{/check-input}}
+        <input type="checkbox" name="checkrow1" aria-labelledby="{{concat table--id '-node-1'}}" aria-label="Details">
```
Comment (mcoker) [inline src/patternfly/components/Table/examples/table-expandable-example.hbs]:
```suggestion
        <input type="checkbox" name="checkrow1" aria-labelledby="{{concat table--id '-node1'}}" aria-label="Details">
```

File: src/patternfly/components/Table/examples/table-expandable-example.hbs [hunk 4]
```diff
@@ -68,7 +68,7 @@
     {{#> table-tr}}
       {{#> table-td table-td--toggle="true" table-td--button--attribute=(concat 'aria-labelledby="' table--id '-node2 expandable-toggle2" id="expandable-toggle2" aria-label="Details" aria-controls="' table--id '-content2"')}}{{/table-td}}
       {{#> table-td table-td--check="true"}}
-        {{#> check-input check-input--attribute=(concat 'type="checkbox" name="checkrow2" aria-labelledby="' table--id '-node2" aria-label="Details"')}}{{/check-input}}
+        <input type="checkbox" name="checkrow2" aria-labelledby="{{concat table--id '-node-2'}}" aria-label="Details">
```
Comment (mcoker) [inline src/patternfly/components/Table/examples/table-expandable-example.hbs]:
```suggestion
        <input type="checkbox" name="checkrow2" aria-labelledby="{{concat table--id '-node2'}}" aria-label="Details">
```

**Comment by mcoker** [comment]
thanks @boaz0, left some suggestions.

**Pull request merged**